variables.tf

```diff
@@ -417,3 +417,85 @@ variable "trusted_profile_roles" {
   description = "List of roles given to the trusted profile."
   default     = ["Administrator"]
 }
+
+########################################################################################################################
+# CBR Variables
+########################################################################################################################
+
+variable "provision_cbr" {
+  type        = bool
+  description = "Set to true to enable creation of CBR Rules and Zones in the module."
+  default     = false
+}
+
+variable "cbr_prefix" {
+  type        = string
+  description = "String to use as the prefix for all CBR resources"
+  default     = "acct-infra-base"
+}
+
+variable "cbr_allow_cos_to_kms" {
+  type        = bool
+  description = "Set rule for COS to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_block_storage_to_kms" {
+  type        = bool
+  description = "Set rule for block storage to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_roks_to_kms" {
+  type        = bool
+  description = "Set rule for ROKS to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_icd_to_kms" {
+  type        = bool
+  description = "Set rule for ICD to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_event_streams_to_kms" {
+  type        = bool
+  description = "Set rule for Event Streams (Messagehub) to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_vpcs_to_container_registry" {
+  type        = bool
+  description = "Set rule for VPCs to container registry, default is true"
+  default     = true
+}
+
+variable "cbr_allow_vpcs_to_cos" {
+  type        = bool
+  description = "Set rule for VPCs to COS, default is true"
```

Comment:
description = "Whether to enable the rule that allows Virtual Private Clouds to access Object Storage. Default is true."